src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -714,6 +715,55 @@ export class SavedObjectsRepository {
     return body;
   }
 
+  /**
+   * Deletes all objects from the provided workspace. It used when deleting a workspace.
+   *
+   * @param {string} workspace
+   * @param options SavedObjectsDeleteByWorkspaceOptions
+   * @returns {promise} - { took, timed_out, total, deleted, batches, version_conflicts, noops, retries, failures }
+   */
+  async deleteByWorkspace(
+    workspace: string,
```

Comment:
Is it taking the workspace name or id as the param ?

Reply:
if it's actually taking the id maybe let's try to name like `workspaceId` or add the detail in the comment

Reply:
It's the id, we use workspace everywhere. workspace name is named differently. If we change it to workspaceId, it may cause more confusion unless we change all to workspaceId.

Reply:
make sense. As we are using workspace everywhere to reference as its id, to make the name consistent, let me add this to parameter comment.